openslides/users/migrations/0004_groups.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-01 14:54
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
```

Comment:
No, it's not, because in the `users/models.py` we don't specify a model `Group`, so OpenSlides uses the default group model from Django. When adding
```
g = apps.get_model('users', 'Group')
```
I got a `LookupError: App 'users' doesn't have a 'group' model`.